fennel/datasets/datasets.py

```diff
@@ -504,7 +519,13 @@ def __init__(self, node: _Node, *args):
         self.node = node
         self.node.out_edges.append(self)
 
-    def aggregate(self, *args, along: Optional[str] = None, **kwargs) -> _Node:
+    def aggregate(
+        self,
+        *args,
+        along: Optional[str] = None,
+        emit_on_window: str = "update",
```

Comment:
Let's call this kwarg just `emit`? Multiword names with underscores look ugly.

Reply:
Fixed